test/functional/plugin/lsp_spec.lua

```diff
@@ -847,25 +847,28 @@ describe('LSP', function()
   end
 
   it('highlight groups', function()
-    eq({'LspDiagnosticsError',
-        'LspDiagnosticsErrorFloating',
-        'LspDiagnosticsErrorSign',
-        'LspDiagnosticsHint',
-        'LspDiagnosticsHintFloating',
-        'LspDiagnosticsHintSign',
-        'LspDiagnosticsInformation',
-        'LspDiagnosticsInformationFloating',
-        'LspDiagnosticsInformationSign',
-        'LspDiagnosticsUnderline',
-        'LspDiagnosticsUnderlineError',
-        'LspDiagnosticsUnderlineHint',
-        'LspDiagnosticsUnderlineInformation',
-        'LspDiagnosticsUnderlineWarning',
-        'LspDiagnosticsWarning',
-        'LspDiagnosticsWarningFloating',
-        'LspDiagnosticsWarningSign',
-      },
-      exec_lua([[require'vim.lsp'; return vim.fn.getcompletion('Lsp', 'highlight')]]))
+    eq({
+      'LspDiagnosticsDefaultError',
+      'LspDiagnosticsDefaultHint',
+      'LspDiagnosticsDefaultInformation',
+      'LspDiagnosticsDefaultWarning',
```

Comment:
nit: name could be less verbose by omitting "Default", which is largely implied by the nature of builtin highlight groups. We should at least not repeat this pattern in the future.